### ISSUE TITLE
[Bug report]: 

### ISSUE THREAD

**Issue by Roytheboy05** [iotaledger/firefly#4016]
### Platform

Desktop

### Version

1.6.3

### Description

assembly  token not showing up in wallet

### Expected behaviour

all tokens show when i open wallet

### Actual behaviour

no assembly token shows 

### Can the issue reliably be reproduced?

Yes

### Operating System

Windows

### Diagnostics

```shell
Version 1.6.3
Language: en
Currency: USD
Node selection: Automatic
Platform: win32
Platform Version: 10.0.22000
Platform Architecture: x64
CPU Count: 8
Total Memory: 16066.0 MB
Free Memory: 11099.5 MB
```

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

Previously staked

### Crash Status

None of the above

### Error Messages

no error msg

### Error Logs

```shell
nothing in error log
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Tuditi** [comment]
Closed due to duplication.

**Comment by MiKeF05** [comment]
This should now be solved with version 1.6.5.
